assets/check

```diff
@@ -147,7 +148,7 @@ if [ -n "$tag_filter" ]; then
       if [ -n "$branch" ]; then
         branch_flag="--merged $branch"
       fi
-      tag=$(git tag --list "$tag_filter" --sort=creatordate $branch_flag | tail -1)
+      tag=$(git tag --list "$tag_filter" --sort=creatordate $branch_flag | tail -$version_depth)
```

Comment:
Does this (and the other change) need to be reversed too?

Reply:
Updated to include the bits under tag_filter. I've also added a new test for them